dotCMS/src/integration-test/java/com/liferay/portal/ejb/UserPersistenceTest.java

```diff
@@ -0,0 +1,219 @@
+package com.liferay.portal.ejb;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.liferay.portal.model.User;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Unit test for the {@link UserPersistence}
+ */
+public class UserPersistenceTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception{
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Test
+    public void test_diff_same_user_nodiff()  {
+
+        final UserPersistence userPersistence = new UserPersistence();
+        final UserHBM userHBM = new UserHBM();
+        final User    user    = new User();
+
+        userHBM.setPassword("xxx");
+        user.setPassword("xxx");
+
+        userHBM.setCompanyId("yyy");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "yyy" appears 15 times in this file; the first occurrence is on line 30](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)